site/templates/origin__3_harryette_mullen.php

```diff
@@ -1,7 +1,64 @@
-<?php snippet('header'); ?>
+<?php snippet('header', ['headerClass' => 'mullen']); ?>
+<div class="navigation__container">
+  <nav>
+    <ul class="navigation">
+      <li class="navigation__item highlight"><a href="<?= $site->url(); ?>"><?= $site->title(); ?></a></li>
```

Comment:
we'll need to add a bio eventually. look at how i did the nav in the newest version of biggers template and the biggers blueprint

Reply:
i'm confused. i deleted the navigation div and am using php to overwrite nav to have the bio show up like biggers but in google inspect it still shows the navigation container (one i deleted..) and my changes are not showing up. .